### ISSUE TITLE
✅ ♿ 🐞 Bug Report: ARIA missing in the new menu widget - [ED-8933]

### ISSUE THREAD

**Issue by kulterryan** [elementor/elementor#23607]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I see the following reported in the `"pagespeed.web.dev"`:

- `[aria-*]` attributes do not match their roles
- `[role]s` are not contained by their required parent element

The accessibility is creating any issue.

![image](https://github.com/elementor/elementor/assets/36154235/cdb65b4b-46b8-4f2c-ab3c-67f87feedf26)

### Steps to reproduce

You can run the `"pagespeed.web.dev"` test or use Lighthouse in the browser to test this.

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>Click to reveal</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.57 (Debian)
	MySQL version: mariadb.org binary distribution v10.6.3
	PHP Version: 8.1.22
	PHP Memory Limit: 511M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 100M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.3
	Site URL: https://cleverviral.co
	Home URL: https://cleverviral.co
	WP Multisite: No
	Max Upload Size: 100 MB
	Memory limit: 768M
	Max Memory limit: 768M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: Asia/Kolkata
	Debug Mode: Inactive

== Theme ==
	Name: ThemeCV
	Version: 2.0.0
	Author: Aryan Chaurasia
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.8.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.62

== Active Plugins ==
	Breeze
		Version: 2.0.29
		Author: Cloudways

	Elementor
		Version: 3.15.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.15.1
		Author: Elementor.com

	JetEngine
		Version: 3.2.4
		Author: Crocoblock

	MalCare WordPress Security Plugin - Malware Scanner, Cleaner, Security Firewall
		Version: 5.24
		Author: MalCare Security

	Perfmatters
		Version: 2.1.4
		Author: forgemedia

	Rank Math SEO
		Version: 1.0.122
		Author: Rank Math

	Yoast Duplicate Post
		Version: 4.5
		Author: Enrico Battocchi & Team Yoast


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimized DOM Output: Inactive
	Improved Asset Loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Inactive
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active by default
	Grid Container: Active
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Active by default
	Editor Top Bar: Inactive
	Landing Pages: Inactive
	Nested Elements: Active
	Lazy Load Background Images: Inactive
	Global Style Guide: Active
	Page Transitions: Active by default
	Notes: Inactive
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Active
	Taxonomy Filter: Active


== Log ==
	
JS: showing 13 of 13JS: 2023-08-08 14:06:17 [error X 1][https://cleverviral.co/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view23419") has already been destroyed and cannot be used. 
JS: 2023-08-08 14:06:17 [error X 1][https://cleverviral.co/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view22193") has already been destroyed and cannot be used. 
JS: 2023-08-08 19:52:35 [error X 1][https://cleverviral.co/wp-admin/post.php?post=726&action=elementor:9:29] Cannot read properties of undefined (reading 'url') 
JS: 2023-08-11 12:55:33 [error X 8][https://cleverviral.co/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.15.2:3:802938] Cannot read properties of undefined (reading 'localized') 
JS: 2023-08-22 13:29:48 [error X 1][https://cleverviral.co/wp-content/plugins/elementor/assets/js/web-cli.min.js?ver=3.15.2:3:107466] Routes: `panel/editor/categories` not found. 
JS: 2023-08-23 09:09:01 [error X 1][https://cleverviral.co/wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=3.15.1:2:64105] Cannot convert undefined or null to object 
JS: 2023-08-23 10:13:07 [error X 2][https://cleverviral.co/wp-content/plugins/elementor/assets/lib/pickr/pickr.min.js?ver=1.5.0:2:19552] Cannot read properties of null (reading 'save') 
JS: 2023-08-24 05:09:48 [error X 79][https://cleverviral.co/wp-content/plugins/elementor-pro/assets/js/lottie.565b778d23c04461c4ea.bundle.min.js:2:2267] bodymovin is not defined 
JS: 2023-08-24 19:22:06 [error X 1][https://cleverviral.co/wp-content/plugins/elementor/assets/js/common-modules.min.js?ver=3.15.2:2:55595] Model.isValidChild() should be implemented, please provide 'isValidChild' functionality. 
JS: 2023-08-25 08:22:36 [error X 1][https://cleverviral.co/wp-content/plugins/jet-engine/assets/js/frontend.js?ver=3.2.4:1732:17] $sliderItems.slick is not a function 
JS: 2023-08-26 18:18:35 [error X 1][https://cleverviral.co/wp-content/plugins/elementor/assets/js/7fada1a61d72da6d46ab.bundle.min.js:2:7826] Cannot read properties of null (reading 'dataset') 
JS: 2023-08-27 11:43:28 [error X 7][https://cleverviral.co/wp-admin/load-scripts.php?c=1&loadchunk_0=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=6.3:2:28722] Cannot read properties of undefined (reading 'value') 
JS: 2023-08-29 16:23:20 [error X 2][https://cleverviral.co/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.15.3:3:884194] Cannot read properties of null (reading 'config') 

PHP: showing 16 of 16PHP: 2023-08-08 19:44:51 [warning X 42][../wp-content/plugins/elementor/includes/base/controls-stack.php::1407] Undefined array key "_element_vertical_align_tablet" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-08-08 23:55:09 [warning X 43][../wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-08-09 16:16:31 [warning X 4][../wp-content/plugins/elementor/modules/history/revisions-manager.php::363] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-08-09 21:01:48 [warning X 75][../wp-content/plugins/elementor/includes/base/controls-stack.php::1407] Undefined array key "_flex_order_tablet" [array (
  'trace' => '
#0: ../wp-content/plugins/elementor/includes/base/controls-stack.php(1407): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ../wp-content/plugins/elementor/core/files/css/base.php(922): Elementor\Controls_Stack -> is_control_visible()
#2: Elementor\Core\Files\CSS\Base -> Elementor\Core\Files\CSS\{closure}()
#3: ../wp-content/plugins/elementor/core/files/css/base.php(927): class type array_reduce()
#4: ../wp-content/plugins/elementor/core/files/css/base.php(934): Elementor\Core\Files\CSS\Base -> get_active_controls()
',
)]
PHP: 2023-08-09 21:01:48 [warning X 150][../wp-content/plugins/elementor/includes/base/controls-stack.php::1407] Undefined array key "_flex_size_tablet" [array (
  'trace' => '
#0: ../wp-content/plugins/elementor/includes/base/controls-stack.php(1407): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ../wp-content/plugins/elementor/core/files/css/base.php(922): Elementor\Controls_Stack -> is_control_visible()
#2: Elementor\Core\Files\CSS\Base -> Elementor\Core\Files\CSS\{closure}()
#3: ../wp-content/plugins/elementor/core/files/css/base.php(927): class type array_reduce()
#4: ../wp-content/plugins/elementor/core/files/css/base.php(934): Elementor\Core\Files\CSS\Base -> get_active_controls()
',
)]
PHP: 2023-08-09 21:01:48 [notice X 71][../wp-content/plugins/elementor-pro/modules/custom-css/module.php::76] trim(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ../wp-content/plugins/elementor-pro/modules/custom-css/module.php(76): class type trim()
#2: ../wp-includes/class-wp-hook.php(310): ElementorPro\Modules\CustomCss\Module -> add_page_settings_css()
#3: ../wp-includes/class-wp-hook.php(334): WP_Hook -> apply_filters()
#4: ../wp-includes/plugin.php(517): WP_Hook -> do_action()
',
)]
PHP: 2023-08-09 23:42:13 [error X 1][../wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::304] Uncaught Error: Call to a member function update_meta() on null in ../wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:304
Stack trace:
#0 ../wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(177): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions()
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions()
#2 ../wp-content/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func()
#3 ../wp-includes/class-wp-hook.php(310): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request()
#4 ../wp-includes/class-wp-hook.php(334): WP_Hook->apply_filters()
#5 ../wp-includes/plugin.php(517): WP_Hook->do_action()
#6 ../wp-admin/admin-ajax.php(188): do_action()
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-08-10 00:00:54 [warning X 2][../wp-content/plugins/elementor/includes/base/controls-stack.php::1407] Undefined array key "image_height_tablet" [array (
  'trace' => '
#0: ../wp-content/plugins/elementor/includes/base/controls-stack.php(1407): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ../wp-content/plugins/elementor/core/files/css/base.php(922): Elementor\Controls_Stack -> is_control_visible()
#2: Elementor\Core\Files\CSS\Base -> Elementor\Core\Files\CSS\{closure}()
#3: ../wp-content/plugins/elementor/core/files/css/base.php(927): class type array_reduce()
#4: ../wp-content/plugins/elementor/core/files/css/base.php(934): Elementor\Core\Files\CSS\Base -> get_active_controls()
',
)]
PHP: 2023-08-11 17:03:18 [warning X 5][../wp-content/plugins/elementor/includes/base/controls-stack.php::1407] Undefined array key "_flex_size_tablet" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-08-11 17:17:08 [warning X 126][../wp-content/plugins/elementor/app/modules/import-export/module.php::738] Attempt to read property "label" on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-08-11 20:59:53 [warning X 1][../wp-content/plugins/elementor-pro/modules/forms/classes/ajax-handler.php::76] Undefined array key "form_id" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-08-11 23:24:12 [error X 1][../wp-content/plugins/elementor-pro/modules/popup/module.php::126] Uncaught Error: Call to undefined method Elementor\Core\Kits\Documents\Kit::save_display_settings_data() in ../wp-content/plugins/elementor-pro/modules/popup/module.php:126
Stack trace:
#0 [internal function]: ElementorPro\Modules\Popup\Module->save_display_settings()
#1 ../wp-content/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func()
#2 ../wp-includes/class-wp-hook.php(310): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request()
#3 ../wp-includes/class-wp-hook.php(334): WP_Hook->apply_filters()
#4 ../wp-includes/plugin.php(517): WP_Hook->do_action()
#5 ../wp-admin/admin-ajax.php(188): do_action()
#6 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-08-12 11:06:29 [notice X 824][../wp-content/plugins/elementor-pro/modules/custom-css/module.php::76] trim(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-08-20 15:53:05 [notice X 1][../wp-content/plugins/elementor/includes/managers/image.php::81] strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-08-20 16:01:36 [warning X 100][../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key "condition_type" [array (
  'trace' => '
#0: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: ../wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2023-08-26 01:05:46 [warning X 2][../wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::171] Undefined array key "editor_post_id" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 13 of 132023-08-09 20:54:58 [info] elementor::elementor_updater Started 
2023-08-09 20:54:58 [info] Elementor/Upgrades - _on_each_version Start  
2023-08-09 20:54:58 [info] Elementor/Upgrades - _on_each_version Finished 
2023-08-09 20:54:58 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.15.1',
  'to' => '3.15.2',
)]
2023-08-12 11:05:25 [info] elementor-pro::elementor_pro_updater Started 
2023-08-12 11:05:25 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-08-12 11:05:25 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-08-12 11:05:25 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.15.0',
  'to' => '3.15.1',
)]
2023-08-26 22:24:50 [info] elementor::elementor_updater Started 
2023-08-26 22:24:50 [info] Elementor/Upgrades - _on_each_version Start  
2023-08-26 22:24:50 [info] Elementor/Upgrades - _on_each_version Finished 
2023-08-26 22:24:50 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.15.2',
  'to' => '3.15.3',
)]
2023-08-26 22:24:50 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.15.2',
  'to' => '3.15.3',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==

````

</details>

**Comment by nicholaszein** [comment]
Hello @kulterryan! 

Thank you for submitting your issue! 🙏 

As @Daveden2 correctly pointed out, this issue will be fixed with the release of Elementor Pro v3.16.0.

Best regards

**Comment by Thozharajan** [comment]
![Edit-Header-new-with-Elementor](https://github.com/elementor/elementor/assets/145122679/9efb2dde-05fa-430c-82e5-fc35d9e3e636)

How to solve this issue . Elementor new version very difficult

**Comment by nicholaszein** [comment]
Hello there @kulterryan!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.16! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
